client/blocks/comment-detail/comment-detail-actions.jsx

```diff
@@ -12,24 +12,27 @@ export const CommentDetailActions = ( {
 	commentIsLiked,
 	commentIsSpam,
 	commentIsTrash,
+	deleteForever,
 	toggleApprove,
 	toggleLike,
 	toggleSpam,
 	toggleTrash,
 	translate,
 } ) =>
 	<div className="comment-detail__actions">
-		<a
-			className={ classNames( 'comment-detail__action-like', { 'is-liked': commentIsLiked } ) }
-			onClick={ toggleLike }
-		>
-			<Gridicon icon={ commentIsLiked ? 'star' : 'star-outline' } />
-			<span>{
-				commentIsLiked
-					? translate( 'Liked' )
-					: translate( 'Like' )
-			}</span>
-		</a>
+		{ ! commentIsSpam && ! commentIsTrash &&
```

Comment:
in the notifications we had to be aware of the fact that some comments don't have all those actions - we have a tri-state device where the presence of the key means it's _likeable_ and the value is whether _it's liked_.

https://github.com/Automattic/notifications-panel/blob/e27c2ce64f086994f9769a79a7bc056309e81d86/src/templates/actions.jsx#L55

could have similar needs here